app/javascript/store/modules/profile/actions.js

```diff
@@ -40,4 +44,20 @@ export default {
         commit(RECOMMENDATIONS_FETCH_ERROR, error);
       });
   },
+
+  [COMPUTE_PROFILE_PR_INFORMATION]({ commit }, {githubUserLogin, monthLimit}) {
+    commit(START_FETCHING_PR_INFORMATION);
+    axios
+      .get(`/api/users/${githubUserLogin}/pull_requests_information`, {
+        params: decamelizeKeys({
+          monthLimit,
+        }),
+      })
+      .then(response => {
+        commit(PROFILE_PR_INFORMATION_RECEIVED, response.data.response.metrics);
+      })
+      .catch(error => {
+        commit(PROFILE_PR_INFORMATION_FETCH_ERROR, error);
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'PROFILE_PR_INFORMATION_FETCH_ERROR' is not defined  no-undef